.github/workflows/tools/daily-status.py

```diff
@@ -1,18 +1,20 @@
 #!/usr/bin/env python3
 from __future__ import annotations
 
-__python_requires__ = "~= 3.8"
+__python_requires__ = ">= 3.8"
```

Comment:
just curious -- why this change?

Reply:
Specifying an upper bound on `python_requires`/`Requires-Python` is considered bad practice.  It doesn't make a difference here, as this is just for human-readable documentation, but the documentation should still reflect best practices.

Reply:
do you remember where such "bad practice" described? 
From my experience if anything should be upper bound is Python version itself with it changing behavior of such rudimentary operations such as `/` while hopping from 2 to 3, and thus leading to not just crashes but to silently producing incorrect results.

Reply:
The Python devs have stated (somewhere) that the Python 3 → Python 4 transition will be more like the 1 → 2 transition than like 2 → 3, i.e., with much more backwards compatibility.  See https://discuss.python.org/t/requires-python-upper-limits/12663 for too much analysis of the problems with upper Python bounds.